roles/openshift_version/tasks/main.yml

```diff
@@ -75,13 +107,18 @@
 
 - fail:
     msg: "No OpenShift version available, please ensure your systems are fully registered and have access to appropriate yum repositories."
-  when: not is_containerized | bool and openshift_version == '0.0'
+  when:
+  - not is_containerized | bool
+  - openshift_version == '0.0'
 
 # We can't map an openshift_release to full rpm version like we can with containers, make sure
 # the rpm version we looked up matches the release requested and error out if not.
 - fail:
     msg: "Detected OpenShift version {{ openshift_version }} does not match requested openshift_release {{ openshift_release }}. You may need to adjust your yum repositories, inventory, or run the appropriate OpenShift upgrade playbook."
-  when: not is_containerized | bool and openshift_release is defined and not openshift_version.startswith(openshift_release) | bool
+  when:
+  - not is_containerized | bool
+  - openshift_release is defined
+  - not openshift_version.startswith(openshift_release) | bool
```

Comment:
Again, this is old code that I just cleaned up. In this case I just put them on multiple lines instead of one long line. I'd have to trace back through the code to answer this. I can do so if you think there is an issue here.